types/jsdom/tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "lib": ["ESNext"],
+        "lib": ["ESNext", "DOM"],
```

Comment:
this should be needed since this line exists: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d561758c4ccf256e156cbff44943f665bb07e2cf/types/jsdom/base.d.ts#L1

Reply:
i had to add this because somehow vscode and the testing script would otherwise error out with "non existing types" locally

Reply:
i dont know what changed, but i just tried with it removed again and it seems to "just work" now

Reply:
done in 235cc3855c29e421203a839c93e6b3c34642038c